ui/controls.js

```diff
@@ -766,8 +778,7 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
    * @export
    */
   isPiPEnabled() {
-    if ('documentPictureInPicture' in window &&
-        this.config_.preferDocumentPictureInPicture) {
+    if (this.shouldUseDocumentPictureInPicture_()) {
```

Comment:
I feel like if we "should" use it would not be part of whether we are currently using it.  For example, the config could change after triggering it.

Could isPipEnabled just be `!!(window.documentPictureInPicture?.window || document.pictureInPictureElement)` ?

Reply:
Done!